Add Database Connection Helper Command (#122)
humanmade/altis-local-server: docs/database.md

> # Interacting with Database

Reviewer: ```suggestion
# Interacting with the Database
```

> - You can log into MySQL and run any query in the default database (`wordpress`) by using:

Reviewer: ```suggestion
You can log into MySQL and run any query in the default database `wordpress` by using:
```

```sh
composer server db
```

> - Use `composer server info` to retrieve MySQL info, connection details and forwarded ports:

Reviewer: ```suggestion
Use `composer server db info` to retrieve the MySQL connection details including the hostname and port:
```

```sh
Root password:  wordpress

Database:       wordpress
User:           wordpress
Password:       wordpress

> Version:        5.7.26-1debian9

Reviewer: Was this at the bottom? Think this just needs updating with the latest output.
